src/components/Cards/card.tsx

```diff
@@ -45,12 +45,21 @@ export default function Card({ props }: { props: CardProps }) {
       role="presentation"
       className="group col-span-1 max-w-xs flex-col overflow-hidden rounded-lg bg-grey-100 shadow-md transition hover:-translate-y-0.5 hover:cursor-pointer hover:bg-grey-50"
     >
-      <Modal setShowPopup={setShowModal} showModal={showModal}>
+      <Modal
+        title={
+          <>
+            Repair ID
+            <h2 className="mt-3">
+              {props.title}
+              <br />
+              <StatusPill status={props.status} />
+            </h2>
+          </>
```

Comment:
The resulting html generated has a h2 nested inside a h3. This is unideal. Could just have the status pill inline